runtime/src/app/app.ts

```diff
@@ -62,15 +64,15 @@ stores.session.subscribe(async value => {
 
 export let prefetching: {
 	href: string;
-	promise: Promise<HydratedTarget>;
+	promise: Promise<any>;
```

Comment:
I'm not sure if this should be `any` because then we lose typing info on line 124 then. `Promise<HydratedTarget>` seemed right to me. was there somewhere that was causing issues?

Reply:
No, true. When I did this I had the impression the promise was only ever waited on for synchronization reasons and the result discarded, but that's not the case. But I'll rebase and this will go away.